src/main/java/ch/njol/skript/lang/function/FunctionReference.java

```diff
@@ -264,8 +265,8 @@ public boolean resetReturnValue() {
 	protected T[] execute(Event e) {
 		// If needed, acquire the function reference
 		if (function == null)
-			function = (Function<? extends T>) Functions.getFunction(functionName);
-		
+			function = (Function<? extends T>) Functions.getFunction(functionName, script);
+
 		if (function == null) { // It might be impossible to resolve functions in some cases!
 			Skript.error("Couldn't resolve call for '" + functionName +
 				"'. Be careful when using functions in 'script load' events!");
```

Comment:
I don't think this is necessary anymore. With the Structure API merged, I don't see any reason why this would be problematic :)
```suggestion
			Skript.error("Couldn't resolve call for '" + functionName + "'.");
```